salt/client/ssh/__init__.py

```diff
@@ -157,17 +157,17 @@
         py_cmd_path=`"$py_cmd" -c \
                    'from __future__ import print_function;
                    import sys; print(sys.executable);'`
-        cmdpath=$(command -v $py_cmd 2>/dev/null || which $py_cmd 2>/dev/null)
+        cmdpath=`command -v $py_cmd 2>/dev/null || which $py_cmd 2>/dev/null`
```

Comment:
The only gotcha I'm aware of here would be that the backticks would strip double-quotes but since we have a statically defined list of single-word strings, we should be just fine. Looks good!